Source/WebKit/Shared/NetworkProcessPreferencesForWebProcess.h

```diff
@@ -30,7 +30,6 @@ namespace WebKit {
 struct NetworkProcessPreferencesForWebProcess {
     bool isWebTransportEnabled { false };
     bool usesSingleWebProcess { false };
-
```

Comment:
Unrelated change, please drop from PR.

Reply:
Looks like this change still needs to be dropped.